### PR TITLE
SDK-2094: Regenerate Propel models

### PR DESCRIPTION
**Pull request by VladislavStrelchenko**
Ticket: https://spryker.atlassian.net/browse/SDK-2094

### CODE REVIEW

File: src/Upgrade/Application/Strategy/Common/PackagePostUpdateHandler/PropelUpdateHandler.php [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            sprintf('%s %s', static::PROJECT_CONSOLE_PATH, static::TRANSFER_GENERATE_COMMAND),
+            sprintf('%s %s', static::PROJECT_CONSOLE_PATH, static::PROPEL_SCHEMA_COPY_COMMAND),
+            sprintf('%s %s', static::PROJECT_CONSOLE_PATH, static::PROPEL_MODEL_BUILD_COMMAND),
+        ];
+
+        foreach ($commandList as $command) {
+            $response = $this->processRunner->run(explode(' ', $command));
+            if ($response->getExitCode()) {
+                $stepsExecutionDto->addOutputMessage($response->getErrorOutput());
+                var_dump($response->getErrorOutput());
```
Comment (pavelmaksimov25) [inline src/Upgrade/Application/Strategy/Common/PackagePostUpdateHandler/PropelUpdateHandler.php:103]: Looks like debug code. Must be removed

File: src/Upgrade/Application/Strategy/Common/PackagePostUpdateHandler/PropelUpdateHandler.php [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+     * @return bool
+     */
+    public function isApplicable(StepsResponseDto $stepsExecutionDto): bool
+    {
+        $composerLockDiffDto = $stepsExecutionDto->getComposerLockDiff();
+        if (!$composerLockDiffDto) {
+            return false;
+        }
+
+        if ($this->hasPropel($composerLockDiffDto->getRequireChanges())) {
```
Comment (pavelmaksimov25) [inline src/Upgrade/Application/Strategy/Common/PackagePostUpdateHandler/PropelUpdateHandler.php:78]: just `return $this->hasPropel($composerLockDiffDto->getRequireChanges())`

**Review by DmytroKlymanSpryker — CHANGES_REQUESTED** [review]
As Pavlo said :)

File: src/Upgrade/Application/Strategy/Common/PackagePostUpdateHandler/PropelUpdateHandler.php (hunk 1 above)
Comment (VladislavStrelchenko) [reply src/Upgrade/Application/Strategy/Common/PackagePostUpdateHandler/PropelUpdateHandler.php:103]: You are right 
thanks

File: src/Upgrade/Application/Strategy/Common/PackagePostUpdateHandler/PropelUpdateHandler.php [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+    /**
+     * @param \Upgrade\Application\Dto\StepsResponseDto $stepsExecutionDto
+     *
+     * @return \Upgrade\Application\Dto\StepsResponseDto
+     */
+    public function handle(StepsResponseDto $stepsExecutionDto): StepsResponseDto
+    {
+        $commandList = [
+            sprintf('rm -rf %s', implode(' ', self::GENERATED_MODEL_PATH_LIST)),
```
Comment (pavelmaksimov25) [inline src/Upgrade/Application/Strategy/Common/PackagePostUpdateHandler/PropelUpdateHandler.php:89]: Can we replace `rm -rf` with this one https://symfony.com/doc/current/components/filesystem.html#remove?
It is more stable and we can control failures in this case.

Comment (VladislavStrelchenko) [reply src/Upgrade/Application/Strategy/Common/PackagePostUpdateHandler/PropelUpdateHandler.php]: ++

**Pull request merged**